File: tests/fixtures/security/implicit_pending_intent/positive.java
package test;

import android.app.PendingIntent;
import android.content.Context;
import android.content.Intent;

class InsecurePendingIntent {
    void schedule(Context context, Intent intent) {
        PendingIntent.getService(context, 0, intent, PendingIntent.FLAG_UPDATE_CURRENT);
    }
}

File: tests/fixtures/security/implicit_pending_intent/positive.kt
package test

import android.app.PendingIntent
import android.content.Context
import android.content.Intent

fun insecurePendingIntent(context: Context, intent: Intent) {
    PendingIntent.getBroadcast(context, 0, intent, PendingIntent.FLAG_UPDATE_CURRENT)
}
